pysat/instruments/pysat_testing2d_xarray.py

```diff
@@ -90,82 +89,100 @@ def load(fnames, tag='', inst_id='', malformed_index=False,
                                      data_range=drange['lt'])
     data['mlt'] = ((epoch_name), mlt)
 
-    # do slt, 20 second offset from mlt
+    # SLT, 20 second offset from `mlt`.
     slt = mm_test.generate_fake_data(time_delta.total_seconds() + 20, uts,
                                      period=iperiod['lt'],
                                      data_range=drange['lt'])
     data['slt'] = ((epoch_name), slt)
 
-    # create a fake satellite longitude, resets every 6240 seconds
-    # sat moves at 360/5820 deg/s, Earth rotates at 360/86400, takes extra time
-    # to go around full longitude
+    # Create a fake satellite longitude, resets every 6240 seconds.
+    # Sat moves at 360/5820 deg/s, Earth rotates at 360/86400, takes extra time
+    # to go around full longitude.
     longitude = mm_test.generate_fake_data(time_delta.total_seconds(), uts,
                                            period=iperiod['lon'],
                                            data_range=drange['lon'])
     data['longitude'] = ((epoch_name), longitude)
 
-    # create fake satellite latitude for testing polar orbits
+    # Create fake satellite latitude for testing polar orbits.
     angle = mm_test.generate_fake_data(time_delta.total_seconds(), uts,
                                        period=iperiod['angle'],
                                        data_range=drange['angle'])
     latitude = max_latitude * np.cos(angle)
     data['latitude'] = ((epoch_name), latitude)
 
-    # create constant altitude at 400 km for a satellite that has yet
-    # to experience orbital decay
+    # Create constant altitude at 400 km for a satellite that has yet
+    # to experience orbital decay.
     alt0 = 400.0
     altitude = alt0 * np.ones(data['latitude'].shape)
     data['altitude'] = ((epoch_name), altitude)
 
-    # create some fake data to support testing of averaging routines
+    # Create some fake data to support testing of averaging routines.
     mlt_int = data['mlt'].astype(int).data
     long_int = (data['longitude'] / 15.).astype(int).data
     data['dummy1'] = ((epoch_name), mlt_int)
     data['dummy2'] = ((epoch_name), long_int)
     data['dummy3'] = ((epoch_name), mlt_int + long_int * 1000.)
     data['dummy4'] = ((epoch_name), uts)
-
-    # Add dummy coords
+    data['string_dummy'] = ((epoch_name),
+                            ['test'] * len(data.indexes[epoch_name]))
+    data['unicode_dummy'] = ((epoch_name),
+                             [u'test'] * len(data.indexes[epoch_name]))
+    data['int8_dummy'] = ((epoch_name),
+                          np.array([1] * len(data.indexes[epoch_name]),
+                          dtype=np.int8))
+    data['int16_dummy'] = ((epoch_name),
+                           np.array([1] * len(data.indexes[epoch_name]),
+                           dtype=np.int16))
+    data['int32_dummy'] = ((epoch_name),
+                           np.array([1] * len(data.indexes[epoch_name]),
```

Comment:
```suggestion
                           np.array([1] * len(data.indexes[epoch_name]),
```

Would be clearer to use `np.ones`